scripts/gen-api-gtkdoc.py

```diff
@@ -396,13 +399,25 @@ def main(args):
     else:
         outfile = sys.stdout
 
+    if (opts.scioutfile):
+        try:
+            scioutfile = open(opts.scioutfile, "w+")
+        except OSError as err:
+            sys.stderr.write("failed to open \"%s\" for writing (%s)\n" % (opts.scioutfile, err.strerror))
+            return 1
+    else:
+        scioutfile = outfile
+
     try:
         outfile.write("/*\n * Automatically generated file - do not edit\n */\n\n")
-        outfile.write("#include <glib.h>\n")
-        outfile.write("#include <gtk/gtk.h>\n\n")
-        outfile.write("typedef struct _ScintillaObject ScintillaObject;\n")
+        outfile.write("#include \"gtkcompat.h\"\n")
+        outfile.write("#include \"sciwrappers.h\"\n")
```

Comment:
Why include sciwrappers here?  wouldn't it make more sense to include `Scintilla.h` and `ScintillaWidget.h`?
Anyway, good point including *gtkcompat.h* for the GtkComboBoxText hack.  Though, now we depend on 2.24 I'd like to break API and ABI and drop the compat here at some point.